app/(general)/integration/pooltogether-v4/layout.tsx

```diff
@@ -0,0 +1,61 @@
+'use client'
+import { ReactNode } from 'react'
+
+import classNames from 'clsx'
+import { motion } from 'framer-motion'
+import Image from 'next/image'
+import Link from 'next/link'
+import Balancer from 'react-wrap-balancer'
+
+import { FADE_DOWN_ANIMATION_VARIANTS } from '@/config/design'
+import { turboIntegrations } from '@/data/turbo-integrations'
+
+export default function PoolTogetherLayout({ children }: { children: ReactNode }) {
+  const classes = classNames('GeneralLayout', 'bg-gradient-dark flex flex-col pb-10 lg:pb-12')
+  const button = classNames(
+    'bg-gradient-pooltogether flex max-w-fit text-xl font-bold items-center justify-center space-x-2 rounded-full px-5 py-2 text-white transition hover:scale-105'
+  )
```

Comment:
When using a single className string there's no need to use `classNames`
```suggestion
  const button =
    'bg-gradient-pooltogether flex max-w-fit text-xl font-bold items-center justify-center space-x-2 rounded-full px-5 py-2 text-white transition hover:scale-105'
```